docs/architecture/cloud-native/infrastructure-resiliency-azure.md

```diff
@@ -73,7 +87,7 @@ We encouraged the best practice of implementing programmatic retry operations in
 
 - *Azure Redis Cache.* The Redis StackExchange client uses a connection manager class that includes retries on failed attempts. The number of retries, specific retry policy and wait time are all configurable.
 
-- *Azure Service Bus.* The Service Bus client exposes a [RetryPolicy class](xref:Microsoft.ServiceBus.RetryPolicy) that can be configured with a back-off interval, retry count, and <xref:Microsoft.ServiceBus.RetryExponential.TerminationTimeBuffer%2A>, which specifies the maximum time an operation can take. The default policy is nine maximum retry attempts with a 30-second backoff period between attempts.
+- *Azure Service Bus.* The Service Bus client exposes a [RetryPolicy class](https://docs.microsoft.com/dotnet/api/microsoft.servicebus.retrypolicy?view=azure-dotnet) that can be configured with a back-off interval and retry count, which specifies the maximum time an operation can take. The default policy is nine maximum retry attempts with a 30-second backoff period between attempts.
```

Comment:
The earlier reference is correct. Please revert to original.

Reply:
Retracted, but not sure the xref does anything? When I click on them, nothing happens.